etc/picongpu/taurus-tud/V100.tpl

```diff
@@ -55,9 +55,10 @@
 # That is, replace in the following line the two appearances of 6 with 3.
 .TBG_gpusPerNode=`if [ $TBG_tasks -gt 6 ] ; then echo 6; else echo $TBG_tasks; fi`
 
-# number of cores to block per GPU - we got 28 cpus per gpu
-#   and we will be accounted 28 CPUs per GPU anyway
-.TBG_coresPerGPU=28
+# number of cores to block per GPU - we got 7 cpus per gpu (44cores/6gpus ~ 7cores)
+# this does not yet include hyperthreading and assumes the default setting
+# #SBATCH --hint=nomultithread
```

Comment:
I find this a bit confusing. Do we now assume that someone (e.g. system administrator) already have set `#SBATCH --hint=nomultithread`? Couldn't we set this by ourselves here?

Reply:
I agree - I will set the default explicitly.